.gitignore

```diff
@@ -0,0 +1 @@
+ocm-cli-shim
```

Comment:
Provide me with an explanation on why this was added, even though it is obvious.

Reply:
<img width="1546" height="263" alt="image" src="https://github.com/user-attachments/assets/aee11c02-5fb6-4aaa-99f6-e0e3173cde2c" />